doc/material-getting-started.md

```diff
@@ -106,25 +139,26 @@ If your application is based on the older solution template that includes a shar
     </Application>
     ```
 
+---
+
 ## Customization
 
 ### Color Overrides using _Material Theme Builder_ and DSP format
 
-It is possible to use the [Material Theme Builder](https://m3.material.io/theme-builder#/custom) to generate a custom color palette derived from your own basic colors. The generated palette is provided in the [DSP format](https://m3.material.io/styles/color/the-color-system/color-dsp) and can be used to override the default Uno.Material colors.
+It is possible to use the [Material Theme Builder](https://m3.material.io/theme-builder#/custom) to generate a custom color palette derived from your own basic colors. The generated palette is provided in the [DSP format](https://m3.material.io/styles/color/the-color-system/color-dsp) and can be used to override the default Uno Material colors.
 
-The tooling required to generate the _Material Colors Override_ file from a DSP package (zip file) will be present by default when creating a _Uno Extensions_ project with support for Uno.Material.
+The tooling required to generate the _Material Colors Override_ file from a DSP package (zip file) will be present by default when creating an Uno Platform application with support for Uno Material from the Wizard.
 
 ![Wizard - Theme Selection](assets/material-theme-selection-wizard.png)
 
 Follow this link to get [more Information about the DSP tooling](xref:Uno.Material.DSP).
```

Comment:
this link `xref:Uno.Material.DSP` is currently not working on the prod and staging site.
cf. my first question to test locally the links that are in the main doc and used in the external doc
![image](https://github.com/unoplatform/Uno.Themes/assets/16295702/842d4a41-bc2d-48a2-a1fc-63a576d38dd9)